### ISSUE TITLE
Allow to delete current feature when editing attributes panel is open

### ISSUE THREAD

**Issue by Raruto** [g3w-suite/g3w-client-plugin-editing#66]
## Motivation

As per **g3w-client-plugin@v3.6.6**,

after editing is activated, the end user must perform at least three steps (in addition to all other map clicks..) before being able to delete an existing feature:

![image](https://github.com/g3w-suite/g3w-client-plugin-editing/assets/9614886/11b43e79-b9db-4a68-b2ff-6d39312d0afa)

## Suggested solution

Add a button somewhere in the UI that allows you to delete the currently displayed feature, i.e. conceptually similar to the following situation:

![image](https://github.com/g3w-suite/g3w-client-plugin-editing/assets/9614886/64add201-b80d-4e43-a94c-9479c705f85e)

**Comment by volterra79** [comment]
@Raruto close button (2) first image is only to close editing form

**Comment by Raruto** [comment]
> close button (2) first image is only to close editing form

You are right, it is used to show the current sequence needed before being able to delete a feature.

Those are usually the steps I take before deciding whether to delete it (ie. does the feature contain useful data to me? --> if so, go from step 2 to 3)

![image](https://github.com/g3w-suite/g3w-client-plugin-editing/assets/9614886/11b43e79-b9db-4a68-b2ff-6d39312d0afa)

**Comment by volterra79** [comment]
@Raruto is some one need to delete a feature, just click on tool delete button (3) and click on element on map.
